post_office/models.py

```diff
@@ -269,6 +269,7 @@ class EmailTemplate(models.Model):
         default='', blank=True)
     default_template = models.ForeignKey('self', related_name='translated_templates',
         null=True, default=None, verbose_name=_('Default template'), on_delete=models.CASCADE)
+    example_context = context_field_class(_('Context'), blank=True, null=True)
```

Comment:
Why are we adding a new field to the model? Adding a preview to admin shouldn't necessitate adding a new field.

Reply:
I think the variable name is self explanatory, but it's to add a context to the rendered preview so all variables can be rendered with a useful value.